core/templates/core/community-create.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'core/css/style.css' %}">
+{% endblock %}
+
+{% block content %}
+<style>
+.hidden {
+    display: none;
+}
+
+</style>
+<script>
```

Comment:
I don't see it being used. If it please point me to the place, if not, please remove.

Reply:
You mean all file is not used?

Reply:
No, I mean this js script.

Reply:
OK, removed. Sorry, I'm not good at frontend and js scripts...

Reply:
Easy, thanks.